src/test/java/com/productdock/library/rental/service/RentalRecordApiTest.java

```diff
@@ -31,7 +31,9 @@ class RentalRecordApiTest extends KafkaTestBase {
 
     public static final String FIRST_BOOK = "1";
     public static final String TEST_FILE = "testRecord.txt";
-    public static final String PATRON = "test@productdock.com";
+    public static final String PATRON_1 = "test@productdock.com";
```

Comment:
use [test1@productdock.com](mailto:test1@productdock.com), [test2@productdock.com](mailto:test2@productdock.com), [test3@productdock.com](mailto:test3@productdock.com) accordingly ..so we avoid possible confusion